<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>800</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Pedestrian Detection</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start detect</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>401</width>
      <height>181</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto'; font-size:12pt; font-weight:200; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Video test01.mp4 loaded.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Starting detection...&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Length: 458 frames&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Positive/All positive detection: 120 / 200&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Correctness: 80 %&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Detection time: 0:20:45&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;5 FPS / 458 frames&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupCHBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
	border: 1px solid gray;
    font-size: 10px;
    font-weight: light;

}
QGroupBox::title
{
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 5 5px;
	background-color:transparent;
}
</string>
    </property>
    <property name="title">
     <string>Convex hull</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="labelSizeCH">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Size:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputCHMinSize">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>155</string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputCHSizeMax">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>155</string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputCHDevMax">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>155</string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputCHDevMin">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>155</string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelDevCH">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Deviation:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupMogBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
	border: 1px solid gray;
    font-size: 10px;
    font-weight: light;

}
QGroupBox::title
{
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 5 5px;
	background-color:transparent;
}
</string>
    </property>
    <property name="title">
     <string>MOG</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="labelThreshMog_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputMogTresh">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>155</string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupHogBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
	border: 1px solid gray;
    font-size: 10px;
    font-weight: light;

}
QGroupBox::title
{
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 5 5px;
	background-color:transparent;
}
</string>
    </property>
    <property name="title">
     <string>HOG</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="labelThreshHog">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputHogTresh">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>155</string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupHogBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
	border: 1px solid gray;
    font-size: 10px;
    font-weight: light;

}
QGroupBox::title
{
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 5 5px;
	background-color:transparent;
}
</string>
    </property>
    <property name="title">
     <string>Video stream</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="text">
      <string>Open 
video</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="text">
      <string>Open 
webcam</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupHogBox_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
	border: 1px solid gray;
    font-size: 10px;
    font-weight: light;

}
QGroupBox::title
{
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 5 5px;
	background-color:transparent;
}
</string>
    </property>
    <property name="title">
     <string>Others</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="labelThreshHog_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Positive frames:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputHogTresh_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>155</string>
     </property>
     <property name="maxLength">
      <number>3</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::LogicalMoveStyle</enum>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelAlgorithm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Algorithm:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Alg1</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Show video frames</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
